README.md

```diff
@@ -115,12 +115,11 @@ The ROS 1 node will start printing the published messages to the console.
 ---
 
 Now we start the ROS 2 listener.
-It is important to use the listener from the `ros1_bridge` package since it uses different QoS parameters than the "normal" `listener`.
 
 ```
 # Shell D:
 . <install-space-with-ros2>/setup.bash
-ros1_bridge_listener
+listener
```

Comment:
https://github.com/ros2/ros1_bridge/pull/37/commits/08881e49b207237b8372539fddaa84df9e77f8e6

although we do now rely on users not installing another executable with the same name, but that will probably be true for most demos